<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Settings</class>
 <widget class="QDialog" name="Settings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>495</width>
    <height>205</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="windowIcon">
   <iconset resource="img.qrc">
    <normaloff>:/img/SETTINGS.png</normaloff>:/img/SETTINGS.png</iconset>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>473</width>
     <height>184</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="mainLayout">
    <property name="leftMargin">
     <number>4</number>
    </property>
    <property name="topMargin">
     <number>2</number>
    </property>
    <property name="rightMargin">
     <number>4</number>
    </property>
    <property name="bottomMargin">
     <number>2</number>
    </property>
    <item>
     <layout class="QHBoxLayout" name="firstRowLayout" stretch="0,0,0">
      <property name="spacing">
       <number>5</number>
      </property>
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <property name="leftMargin">
       <number>4</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="watchDirLabel">
        <property name="font">
         <font>
          <family>Courier New</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Директория для мониторинга:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="watchDirLE">
        <property name="font">
         <font>
          <family>Courier New</family>
          <pointsize>10</pointsize>
         </font>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="watchDirButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>23</width>
          <height>23</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>23</width>
          <height>23</height>
         </size>
        </property>
        <property name="sizeIncrement">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="img.qrc">
          <normaloff>:/img/FOLDER.png</normaloff>:/img/FOLDER.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="leftMargin">
       <number>4</number>
      </property>
      <item>
       <widget class="QLabel" name="tempDirLabel">
        <property name="font">
         <font>
          <family>Courier New</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Директория для временных файлов:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="investigatorDirLE">
        <property name="font">
         <font>
          <family>Courier New</family>
          <pointsize>10</pointsize>
         </font>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="tempDirButton">
        <property name="minimumSize">
         <size>
          <width>23</width>
          <height>23</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>23</width>
          <height>23</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="img.qrc">
          <normaloff>:/img/FOLDER.png</normaloff>:/img/FOLDER.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="leftMargin">
       <number>4</number>
      </property>
      <item>
       <widget class="QLabel" name="cleanDirLabel">
        <property name="font">
         <font>
          <family>Courier New</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Директория для чистых файлов:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="cleanDirLE">
        <property name="font">
         <font>
          <family>Courier New</family>
          <pointsize>10</pointsize>
         </font>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="cleanDirButton">
        <property name="minimumSize">
         <size>
          <width>23</width>
          <height>23</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>23</width>
          <height>23</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="img.qrc">
          <normaloff>:/img/FOLDER.png</normaloff>:/img/FOLDER.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <property name="leftMargin">
       <number>4</number>
      </property>
      <item>
       <widget class="QLabel" name="dangerousDirLabel">
        <property name="font">
         <font>
          <family>Courier New</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Директория для зараженных файлов:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="dangerousDirLE">
        <property name="font">
         <font>
          <family>Courier New</family>
          <pointsize>10</pointsize>
         </font>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="dangerousDirButton">
        <property name="minimumSize">
         <size>
          <width>23</width>
          <height>23</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>23</width>
          <height>23</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="img.qrc">
          <normaloff>:/img/FOLDER.png</normaloff>:/img/FOLDER.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="kasperskyLayout">
      <item>
       <widget class="QCheckBox" name="kasperCB">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="kasperFileLabel">
        <property name="minimumSize">
         <size>
          <width>56</width>
          <height>23</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>1111</width>
          <height>23</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Courier New</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Использовать Kaspersky:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="kasperFileLE">
        <property name="font">
         <font>
          <family>Courier New</family>
          <pointsize>10</pointsize>
         </font>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="kasperFileButton">
        <property name="minimumSize">
         <size>
          <width>23</width>
          <height>23</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>23</width>
          <height>23</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="img.qrc">
          <normaloff>:/img/KASPERSKY.png</normaloff>:/img/KASPERSKY.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="drwebLayout">
      <item>
       <widget class="QCheckBox" name="drwebCB">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="drwebFileLabel">
        <property name="minimumSize">
         <size>
          <width>48</width>
          <height>23</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>1111</width>
          <height>23</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Courier New</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Использовать DrWeb:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="drwebFileLE">
        <property name="font">
         <font>
          <family>Courier New</family>
          <pointsize>10</pointsize>
         </font>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="drwebFileButton">
        <property name="minimumSize">
         <size>
          <width>23</width>
          <height>23</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>23</width>
          <height>23</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="img.qrc">
          <normaloff>:/img/DRWEB.jpg</normaloff>:/img/DRWEB.jpg</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="img.qrc"/>
 </resources>
 <connections/>
</ui>
